lib/cdn-site-hosting/cdn-site-hosting-construct.ts

```diff
@@ -35,18 +37,10 @@ export class CdnSiteHostingConstruct extends cdk.Construct {
 
     const siteDomain = getSiteDomain(props);
 
-    // certificate
-    const viewerCertificate = cloudfront.ViewerCertificate.fromAcmCertificate(
-      certificatemanager.Certificate.fromCertificateArn(
-        this,
-        `${siteDomain}-cert`,
-        props.certificateArn
-      ),
-      {
-        aliases: [siteDomain],
-        sslMethod: cloudfront.SSLMethod.SNI,
-        securityPolicy: cloudfront.SecurityPolicyProtocol.TLS_V1_1_2016,
```

Comment:
question: Has this been renamed to `minimumProtocolVersion` below?

Reply:
Yup.